R/tufish_ebook.R

```diff
@@ -0,0 +1,86 @@
+
+#' Tuftish e-Book format (PDF)
+#' 
+#' Template for creating an e-book based on the style of 
+#' Edward R. Tufte and Richard Feynman. 
+#' 
+#' @inheritParams pdf_document
+#' 
+#' @export
+tuftish_ebook <- function(fig_width = 4,
```

Comment:
We should order these arguments the same as pdf_document (e.g. toc args go first)